File: SDKs/motaword.com/1.0/python/src/sdk/models/shared/languagestringstat.py
import dataclasses
from typing import Optional
from dataclasses_json import dataclass_json
from sdk import utils


@dataclass_json
@dataclasses.dataclass
class LanguageStringStat:
    project_string_count: Optional[int] = dataclasses.field(default=None, metadata={'dataclasses_json': { 'letter_case': utils.field_name('project_string_count') }})
    source_language: Optional[str] = dataclasses.field(default=None, metadata={'dataclasses_json': { 'letter_case': utils.field_name('source_language') }})
    tm_string_count: Optional[int] = dataclasses.field(default=None, metadata={'dataclasses_json': { 'letter_case': utils.field_name('tm_string_count') }})
    

File: SDKs/motaword.com/1.0/typescript/src/sdk/models/shared/languagestringstat.ts
import { SpeakeasyMetadata, SpeakeasyBase } from "../../../internal/utils";



export class LanguageStringStat extends SpeakeasyBase {
  @SpeakeasyMetadata({ data: "json, name=project_string_count" })
  projectStringCount?: number;

  @SpeakeasyMetadata({ data: "json, name=source_language" })
  sourceLanguage?: string;

  @SpeakeasyMetadata({ data: "json, name=tm_string_count" })
  tmStringCount?: number;
}
